tests/system/action/user/test_update.py

```diff
@@ -76,10 +82,24 @@ def test_update_template_fields(self) -> None:
                 "structure_level_$": {1: "level_1", 2: "level_2"},
                 "about_me_$": {1: "<p>about</p><iframe></iframe>"},
                 "vote_weight_$": {1: "1.000000", 2: "2.333333"},
+                "committee_$_management_level": {
+                    1: None,
+                    2: CommitteeManagementLevel.CAN_MANAGE,
+                },
+                "committee_ids": [2],
             },
         )
         self.assert_status_code(response, 200)
         user = self.get_model("user/223")
+        assert (
+            CommitteeManagementLevel(user.get("committee_$1_management_level"))
+            == CommitteeManagementLevel.NO_RIGHT
```

Comment:
Please assert explicitly that `user.get("committee_$1_management_level") is None` since the constructor will convert everything not matching any level to `NO_RIGHT`

Reply:
Done